src/backend/distributed/commands/table.c

```diff
@@ -769,6 +805,17 @@ GenerateConstraintName(const char *tabname, Oid namespaceId, Constraint *constra
 			break;
 		}
 
+		case CONSTR_FOREIGN:
+		{
+			conname = ChooseConstraintName(tabname,
```

Comment:
not this PR's fault, but we generally avoid shorthand/acronyms, so "tableName" instead of "tabname".